metadata.rb

```diff
@@ -1,10 +1,10 @@
 name              'java'
 maintainer        'Agile Orbit'
 maintainer_email  'info@agileorbit.com'
-license           'Apache 2.0'
+license           'Apache-2.0'
 description       'Installs Java runtime.'
 long_description  IO.read(File.join(File.dirname(__FILE__), 'README.md'))
-version           '1.50.0'
+version           '1.51.0'
```

Comment:
this feels like maybe it's a major release with the cookbook dependency changes potentially and because of a lot of changes happening even if it's just refactoring a bunch.